dispatcher/backend/docs/openapi_v1.yaml

```diff
@@ -1860,6 +1888,49 @@ components:
       type: integer
       format: int32
       description: Disk space (in bytes) required to run it
+    ImageTags:
```

Comment:
This type `object` is useless if it's not defined. Having this with an overly complex example prevents the developer from making any assumption about the returned data… while it's exactly the objective of the documentation.

This description should be accurate which means we also have to maintain it so that's why I suggest the API returns only what we need (the name)